cmd/util/client.go

```diff
@@ -33,6 +35,10 @@ func GetConfig() (*rest.Config, error) {
 
 // GetClient creates a controller-runtime client for Kubernetes
 func GetClient() (crclient.Client, error) {
+	if os.Getenv("FAKE_CLIENT") == "true" {
```

Comment:
I wonder if the client should be a param of createCluster instead?

Reply:
that would require a bigger refactor, since GetClient() is called in multiple places across the cmd code. I can do that if you think that's better.